package/features/viz.feature

```diff
@@ -2,9 +2,25 @@ Feature: Viz plugin in new project
     Background:
         Given I have prepared a config file with example code
 
+    Scenario: Execute viz with the earliest Kedro version that it supports 
+        Given I have installed kedro version "0.17.5"
+        And I have run a non-interactive kedro new with pandas-iris starter
+        And I have installed the project's requirements
+        When I execute the kedro viz command
+        Then kedro-viz should start successfully
+
+    Scenario: Execute viz with latest Kedro with lower-bound viz requirements
```

Comment:
@merelcht 
this is because in our last release we had an issue where strawberry-graphql below < 0.190 wouldn't work with Kedro-viz and we were not able to capture that in our manual testing. 

So we decided to have an e2e-test that would check if kedro-viz works with the lower-bound of all our dependencies (this would mean some maintenance work but we don't have so many dependencies) and if it didn't we know we need to update these dependencies.

Reply:
Okay cool, that's very useful 👍